src/librustdoc/clean/inline.rs

```diff
@@ -510,7 +510,7 @@ fn build_static(cx: &DocContext<'_>, did: DefId, mutable: bool) -> clean::Static
     clean::Static {
         type_: cx.tcx.type_of(did).clean(cx),
         mutability: if mutable { Mutability::Mut } else { Mutability::Not },
-        expr: "\n\n\n".to_string(), // trigger the "[definition]" links
```

Comment:
What did this comment mean? What is a `[definition]` link?

Reply:
I honestly have no idea and I didn't go through the git history to discover it. Some mysteries are better when they remain mysterious. :)

Reply:
The comment was added in 06f3f9a0c9f25c92b7362a4af6067df8902bd057. At that point in time https://github.com/rust-lang/rust/blob/06f3f9a0c9f25c92b7362a4af6067df8902bd057/src/librustdoc/html/render.rs#L1431 looks the most relevant. That was removed in 55af4aff5641232e5c08ecf2402b1c795a24e50b, which merged in https://github.com/rust-lang/rust/pull/19234 and no longer checks for newlines.

The current output for statics looks like this:  ![image](https://user-images.githubusercontent.com/23638587/107696588-29554580-6c80-11eb-8328-c7879fdfa262.png)
Can you add a test to make sure it stays the same after this change?

Reply:
Sure!

Reply:
There is https://github.com/rust-lang/rust/blob/master/src/test/rustdoc/static.rs already. I think it's enough but do you see something to add maybe?

Reply:
Looks great, thanks!